automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/monitor/SubmissionDetailsPage.scala

```diff
@@ -63,16 +63,24 @@ class SubmissionDetailsPage(namespace: String, name: String, var submissionId: S
     statusMessage.getText
   }
 
+  def verifyWorkflowStatus(expectedStatus: String): Boolean = {
+    val actualStatus = readWorkflowStatus()
+    val pass = expectedStatus.contains(actualStatus)
```

Comment:
why `contains` here and not `equals`?  I'm copying the logic I refactored out of  `verifyWorkflowSucceeded`/`verifyWorkflowFailed`/`verifyWorkflowAborted`  so as not to change anything.